python/CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ cd beeai-framework
 cd python
 
 # Use Poetry to install the project dependencies and activate a virtual environment
-poetry install
+poetry install # poetry install --all-extras
```

Comment:
Given the mypy problems we saw in #576 should all extras be on a separate line so it appears mandatory?

The way it currently is makes it appear optional to me.

Reply:
I did it this was since that was how we did `poe format --fix above` and not everyone following these step plan to contribute code, some just want a local build to play with the cutting edge. But I'm ok with just replacing `poetry install` with `poetry install --all-extras entirely here